generator/mavgen_python.py

```diff
@@ -1135,11 +1138,10 @@ def generate(basename, xml):
         filename = basename + ".py"
 
     msgs = []
-    enums = []
+    enums = xml[0].enum #All enums merged in top level
```

Comment:
So previously Python built up all enums by extending the enums at each level.
This essentially seemed to work.

But now that the top level has all the enums we can just include that from the top level file we are building. Shown to work.